workflow-trace-viewer/src/jvmMain/kotlin/com/squareup/workflow1/traceviewer/util/UploadFile.kt

```diff
@@ -35,12 +36,14 @@ public fun UploadFile(
       type = FileKitType.File(listOf("json", "txt")),
       title = "Select Workflow Trace File"
     ) {
+      onReset()
       onFileSelect(it)
     }
+
     Button(
       onClick = { launcher.launch() },
       modifier = Modifier
-        .align(Alignment.BottomEnd),
+        .align(Alignment.BottomStart),
```

Comment:
I noticed that there is a comment in your App method saying that this is a bottom left button, but if you instead passed set the alignment of the button in the modifier you're already passing in then the comment wouldn't be required. I also think it's more typical that the parent would specify the position of a button rather than the button itself specifying where it is on screen.